docs/tutorials/hera_sim_defaults.ipynb

```diff
@@ -18,36 +18,15 @@
    "cell_type": "code",
```

Comment:
<p>While not necessary, I would like to update this to make better plots in the future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Noted.</p>